src/style.css

```diff
@@ -1,3 +1,81 @@
 body {
   margin: 0;
+  padding: 0;
+}
+
+html {
+  font-family: Roboto, Arial, sans-serif;
+  font-size: 12px;
+  font-weight: 500;
+
+  --active-color: #00acdc;
+}
+
+.header {
+  display: flex;
+  width: 100%;
+  align-items: center;
+  justify-content: space-between;
+}
+
+.header__nav {
+  margin: 0 50px 0 0;
+}
+
+.header__list {
+  display: flex;
+  margin: 0;
+  padding: 0;
+  list-style: none;
+}
+
+.header__link {
+  display: flex;
+  height: 60px;
+  line-height: 60px;
+
+  color: #000;
+  text-decoration: none;
+  text-transform: uppercase;
+}
+
+li {
+  margin-right: 20px;
+}
```

Comment:
Avoid using tag names for styling (except `html` and `body`). Instead, style elements using `.class`. For example, consider adding a class to the `li` elements within `.header__list` and style that class instead.